python_src/src/lamp_py/performance_manager/pipeline.py

```diff
@@ -59,6 +70,8 @@ def main(args: argparse.Namespace) -> None:
         db_index=DatabaseIndex.METADATA, verbose=args.verbose
     )
 
+    run_on_app_start()
+
```

Comment:
this _could_ go in `start()` but i don't think it matters.

Reply:
Incorporated this, also bumped up the hours per event loop to 12, to hopefully speed up processing of RT data